en/advanced/system_startup.md

```diff
@@ -1,10 +1,38 @@
 # System Startup
 
-The PX4 boot is controlled by shell scripts in the [ROMFS/px4fmu_common/init.d](https://github.com/PX4/Firmware/tree/master/ROMFS/px4fmu_common/init.d) folder.
+The PX4 startup is controlled by shell scripts.
+On NuttX they reside in the [ROMFS/px4fmu_common/init.d](https://github.com/PX4/Firmware/tree/master/ROMFS/px4fmu_common/init.d) folder - some of these are also used on Posix (Linux/MacOS). The scripts that are only used on Posix are located in [ROMFS/px4fmu_common/init.d-posix](https://github.com/PX4/Firmware/tree/master/ROMFS/px4fmu_common/init.d-posix).
```

Comment:
I changed to "used on Posix **(Linux/MacOS)**" - ie to add the meaning of posix first time it is used. Slightly leaning towards referring this as "Simulation"

Reply:
This is good. It's not only simulation, as in future our Linux-based boards (such as RPi) should use that as well.